libs/core/web/src/lib/pages/film-list/FilmList.tsx

```diff
@@ -0,0 +1,33 @@
+import { i18nKeys } from '@form-exercise/i18n';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import { useTranslation } from 'react-i18next';
+import { SmallInfoCard } from '@form-exercise/ui';
+import { useFetchOmdb } from '@form-exercise/utils';
+import { Film } from '@form-exercise/data/interface';
+
+export function FilmList() {
+  const { t } = useTranslation();
+  const { isLoading, data: films } = useFetchOmdb({ searchBy: 'Batman' });
+
+  return (
+    <>
+      <Typography variant="h2">{t(i18nKeys.title.filmList)}</Typography>
+      {!isLoading && (
```

Comment:
it would be better to still render something when it's loading, a skeleton of the films grid or a loader